src/vs/workbench/contrib/terminalContrib/chatAgentTools/browser/tools/monitoring/outputMonitor.ts

```diff
@@ -45,6 +45,13 @@ export class OutputMonitor extends Disposable implements IOutputMonitor {
 
 	private _lastAutoReply: string | undefined;
 
+	// Telemetry counters
+	private _inputToolManualAcceptCount = 0;
+	private _inputToolManualRejectCount = 0;
+	private _inputToolManualChars = 0;
+	private _inputToolSuggestChars = 0;
+	private _inputToolSuggestInputCount = 0;
```

Comment:
Could we expose this as a single `inputToolCounters` property that includes everything? That way we can also pull these directly off `outputMonitor` instead of adding to the complex return types.

Reply:
can do when we expose these later